app/assets/javascripts/components/master_overlay_panel/master_overlay_panel_group.js.coffee

```diff
@@ -14,7 +14,7 @@ do ($=jQuery, ko) ->
         primaryTitle = ko.observable('')
         secondaryTitle = ko.observable('')
         shouldAnimate = ko.observable(true)
-        indicatorElement = ko.observable(params.indicatorElement)
+        state = params.state
```

Comment:
Is this used anywhere?

Reply:
Ah, good catch. It is a left over. I’ll remove it.